js/index.js

```diff
@@ -21,6 +21,10 @@ for (let i = 0; i < numOfMembers; i++) {
   memberLinks.appendChild(memberTags);
   memberSection.appendChild(memberLinks);
 }
+const joinButtonDisable = document.getElementById('button-disable');
+joinButtonDisable.addEventListener('click', (event) => {
+  event.preventDefault();
+});
```

Comment:
Hey Why you are doing this all??
https://www.w3schools.com/tags/att_button_disabled.asp
Please check above link

Using the disabled attribute on a <button> tag in HTML has several benefits, mainly related to user experience and accessibility. When you add the disabled attribute to a button element, it prevents users from interacting with or clicking the button.

Reply:
Disabled is working for button .But it is not working anchor tag. So when we apply disabled on button ,it is still clickable .
That's why we need to add JS code